app/.gitignore

```diff
@@ -13,10 +13,17 @@
 
 # Ignore Drupal's file directory
 # /web/sites/*/files/
+/config/sync/*
```

Comment:
Popping in to comment here instead of on https://github.com/18F/osc-website-pa/pull/42/files#r244359574:

You'll want to include (exported) configuration in your codebase somehow, to enable automated deployment. `/config/sync/` is Drupal 8's built-in way of handling that, but it has its limitations (perhaps most notably with regard to content, which it basically ignores). Makes sense to hold the larger config management conversation for another PR, but one way or another, you'll eventually want to include config files.